solc/CommandLineInterface.cpp

```diff
@@ -1195,6 +1198,10 @@ void CommandLineInterface::handleCModelHeaders(
 		cov.print(_os);
 	}
 	for (auto const& ast : _asts)
+	{
+		modelcheck::MainFunction_1(*ast, _con, true).print(_os);
```

Comment:
As is, this would crash and burn if we had more than one contract. I can think of two solutions.
1. We can throw an error if it detects more than 1 contract (this would be within `MainFunction_1`)
2. We could iterate over all `n` contracts and generate `n` different files, each with a main (`contract_A_main.c`, `contract_B_main.c`, ...)
Eventually, we will need support for multiple contracts in a single model, but we aren't there yet, so these solutions should suffice.